### DISCUSSION TITLE
Opening a project for the folder view?

### DISCUSSION THREAD

**Discussion by Fatebreak** [marcusolsson/obsidian-projects#360] [Help] [answered]
Is it possible to turn a project into a note similar to how Database Folder works? If not could that become a feature? I prefer navigating with my folders and would like to be able to click a specific note to open a specific project.

**Comment by marcusolsson** [comment, marked as answer]
Unfortunately no, but there are a couple of related feature requests (#77 and #355). Feel free to add your thoughts or support if you think they could help.
